src/main/java/org/jenkinsci/plugins/configfiles/folder/FolderConfigFileAction.java

```diff
@@ -218,7 +218,10 @@ public FormValidation doCheckConfigId(@QueryParameter("configId") String configI
     }
 
     private void checkPermission(Permission permission) {
-        Hudson.getInstance().checkPermission(permission);
+//        Ancestor ancestor = req.findAncestor(Folder.class);
+//        Folder parent = (Folder) ancestor.getObject();
```

Comment:
yes, sorry, forgot to clean up a bit... shall I or will you? not super active on here ;)